libs/scripts/scripts.js

```diff
@@ -21,103 +21,103 @@ const prodDomains = ['milo.adobe.com'];
-  lu_de: { ietf: 'de-LU', tk: 'vin7zsi.css' },
-  lu_en: { ietf: 'en-LU', tk: 'pps7abe.css' },
-  lu_fr: { ietf: 'fr-LU', tk: 'vrk5vyv.css' },
-  hu: { ietf: 'hu-HU', tk: 'aaz7dvd.css' },
-  mt: { ietf: 'en-MT', tk: 'pps7abe.css' },
-  mena_en: { ietf: 'en', tk: 'pps7abe.css' },
-  nl: { ietf: 'nl-NL', tk: 'cya6bri.css' },
-  no: { ietf: 'no-NO', tk: 'aaz7dvd.css' },
-  pl: { ietf: 'pl-PL', tk: 'aaz7dvd.css' },
-  pt: { ietf: 'pt-PT', tk: 'inq1xob.css' },
-  ro: { ietf: 'ro-RO', tk: 'aaz7dvd.css' },
-  sa_en: { ietf: 'en', tk: 'pps7abe.css' },
-  ch_de: { ietf: 'de-CH', tk: 'vin7zsi.css' },
-  si: { ietf: 'sl-SI', tk: 'aaz7dvd.css' },
-  sk: { ietf: 'sk-SK', tk: 'aaz7dvd.css' },
-  ch_fr: { ietf: 'fr-CH', tk: 'vrk5vyv.css' },
-  fi: { ietf: 'fi-FI', tk: 'aaz7dvd.css' },
-  se: { ietf: 'sv-SE', tk: 'fpk1pcd.css' },
-  ch_it: { ietf: 'it-CH', tk: 'bbf5pok.css' },
-  tr: { ietf: 'tr-TR', tk: 'aaz7dvd.css' },
-  ae_en: { ietf: 'en', tk: 'pps7abe.css' },
-  uk: { ietf: 'en-GB', tk: 'pps7abe.css' },
-  at: { ietf: 'de-AT', tk: 'vin7zsi.css' },
-  cz: { ietf: 'cs-CZ', tk: 'aaz7dvd.css' },
-  bg: { ietf: 'bg-BG', tk: 'aaz7dvd.css' },
-  ru: { ietf: 'ru-RU', tk: 'aaz7dvd.css' },
-  ua: { ietf: 'uk-UA', tk: 'aaz7dvd.css' },
-  il_he: { ietf: 'he', tk: 'nwq1mna.css', dir: 'rtl' },
-  ae_ar: { ietf: 'ar', tk: 'nwq1mna.css', dir: 'rtl' },
-  mena_ar: { ietf: 'ar', tk: 'dis2dpj.css', dir: 'rtl' },
-  sa_ar: { ietf: 'ar', tk: 'nwq1mna.css', dir: 'rtl' },
+  africa: { ietf: 'en', lang: 'en', reg: '002', tk: 'pps7abe.css' },
+  be_fr: { ietf: 'fr-BE', lang: 'fr', reg: 'BE', tk: 'vrk5vyv.css' },
+  be_en: { ietf: 'en-BE', lang: 'en', reg: 'BE', tk: 'pps7abe.css' },
+  be_nl: { ietf: 'nl-BE', lang: 'nl', reg: 'BE', tk: 'cya6bri.css' },
+  cy_en: { ietf: 'en-CY', lang: 'en', reg: 'CY', tk: 'pps7abe.css' },
+  dk: { ietf: 'da-DK', lang: 'da', reg: 'DK', tk: 'aaz7dvd.css' },
+  de: { ietf: 'de-DE', lang: 'de', reg: 'DE', tk: 'vin7zsi.css' },
+  ee: { ietf: 'et-EE', lang: 'et', reg: 'EE', tk: 'aaz7dvd.css' },
+  es: { ietf: 'es-ES', lang: 'es', reg: 'ES', tk: 'oln4yqj.css' },
+  fr: { ietf: 'fr-FR', lang: 'fr', reg: 'FR', tk: 'vrk5vyv.css' },
+  gr_en: { ietf: 'en-GR', lang: 'en', reg: 'GR', tk: 'pps7abe.css' },
+  ie: { ietf: 'en-GB', lang: 'en', reg: 'IE', tk: 'pps7abe.css' },
+  il_en: { ietf: 'en-IL', lang: 'en', reg: 'IL', tk: 'pps7abe.css' },
+  it: { ietf: 'it-IT', lang: 'it', reg: 'IT', tk: 'bbf5pok.css' },
+  lv: { ietf: 'lv-LV', lang: 'lv', reg: 'LV', tk: 'aaz7dvd.css' },
+  lt: { ietf: 'lt-LT', lang: 'lt', reg: 'LT', tk: 'aaz7dvd.css' },
+  lu_de: { ietf: 'de-LU', lang: 'de', reg: 'LU', tk: 'vin7zsi.css' },
+  lu_en: { ietf: 'en-LU', lang: 'en', reg: 'LU', tk: 'pps7abe.css' },
+  lu_fr: { ietf: 'fr-LU', lang: 'fr', reg: 'LU', tk: 'vrk5vyv.css' },
+  hu: { ietf: 'hu-HU', lang: 'hu', reg: 'HU', tk: 'aaz7dvd.css' },
+  mt: { ietf: 'en-MT', lang: 'en', reg: 'MT', tk: 'pps7abe.css' },
+  mena_en: { ietf: 'en', lang: 'en', reg: '', tk: 'pps7abe.css' },
```

Comment:
What's the reasoning behind leaving `mena_en` and `mena_ar` without a region? I think in the past we had `mena`.

Reply:
TBD. Current `ietf` codes are just the lang (ar). I did not change these. `mena` is not a valid region and there isn't a valid code to represent MENA either. Northern Africa is `015` but there is no Middle East region -- those countries are part of Western Asia (`145`).